testutils/src/main/java/com/amplifyframework/testutils/sync/SynchronousAuth.java

```diff
@@ -125,7 +113,7 @@ public static SynchronousAuth delegatingToCognito(Context context, Plugin<?> aut
         }
         //TODO: make authCategory confiuration synchronous
         Thread.sleep(AUTH_OPERATION_TIMEOUT_MS);
```

Comment:
Do we still need this Sleep here?

Reply:
Yes, since initialization is asynchronous too.